homeassistant/components/cover/__init__.py

```diff
@@ -223,14 +225,39 @@ def current_cover_tilt_position(self):
         pass
 
     @property
-    def state(self):
-        """Return the state of the cover."""
-        closed = self.is_closed
+    def cover_state(self):
+        """Return current state of cover."""
+        pass
 
-        if closed is None:
-            return STATE_UNKNOWN
+    @property
+    def cover_state_list(self):
+        """Return list of posible door states."""
+        pass
 
-        return STATE_CLOSED if closed else STATE_OPEN
+    @property
+    def state(self):
+        """Return the state of the cover."""
+        type = self.device_class
+        if type == "garage":
```

Comment:
We still want to keep the same state logic here for garage doors, at least for now. Breaking the current binary state options would be a much larger change than adding a new attribute.